core/arch/arm/plat-hikey/conf.mk

```diff
@@ -7,6 +7,8 @@ core_arm32-platform-aflags	+= -mfpu=neon
 $(call force,CFG_GENERIC_BOOT,y)
 $(call force,CFG_HWSUPP_MEM_PERM_PXN,y)
 $(call force,CFG_PL011,y)
+$(call force,CFG_PL061,y)
+$(call force,CFG_PL022,y)
```

Comment:
That should be:
```
CFG_PL061 ?= y
CFG_PL022 ?= y
```
...at the end of the file. Because you don't want to force the values, i.e., 'make PLATFORM=hikey CFG_PL061=n CFG_PL022=n' should be allowed.